agent/mibgroup/examples/netSnmpHostsTable.c

```diff
@@ -65,8 +65,15 @@ initialize_table_netSnmpHostsTable(void)
     if (!my_handler || !table_info || !iinfo) {
         snmp_log(LOG_ERR,
                  "malloc failed in initialize_table_netSnmpHostsTable");
-        free(iinfo);
-        free(table_info);
+        if (my_handler) {
+            free(my_handler);
+        }
+        if (iinfo) {
```

Comment:
Please leave out the if-tests around the free() calls. From https://pubs.opengroup.org/onlinepubs/007904975/functions/free.html: _If ptr is a null pointer, no action shall occur._